**Mann-Whitney U Test**

Mann-Whitney U test is the non-parametric alternative test to the independent sample t-test.  It is a non-parametric test that is used to compare two sample means that come from the same population, and used to test whether two sample means are equal or not.  Usually, the Mann-Whitney U test is used when the data is ordinal or when the assumptions of the t-test are not met.

**Assumptions of the Mann-Whitney:**

Mann-Whitney U test is a non-parametric test, so it does not assume any assumptions related to the distribution of scores.  There are, however, some assumptions that are assumed
1. The sample drawn from the population is random.
2. Independence within the samples and mutual independence is assumed.  That means that an observation is in one group or the other (it cannot be in both).
3. Ordinal measurement scale is assumed.

## a)

**H0** = There is NO diffrence between ranks of each group (both groups are same)
 
**Ha** = There is a diffrence between ranks of each group (groups are diffrent)

## b)

In [29]:
CE_IQ = c(125,127,126,126,105,128,127,126,79,124)
CE_IQ

[1] 125 127 126 126 105 128 127 126  79 124

In [30]:
EE_IQ = c(131,129,77,52,134,133,94,68,67,132)
EE_IQ

[1] 131 129  77  52 134 133  94  68  67 132

In [31]:
total = c(CE_IQ, EE_IQ)
total

[1] 125 127 126 126 105 128 127 126  79 124 131 129  77  52 134 133  94  68  67
[20] 132

In [32]:
total_sorted = sort(total, decreasing = FALSE)
total_sorted

[1]  52  67  68  77  79  94 105 124 125 126 126 126 127 127 128 129 131 132 133
[20] 134

In [33]:
for (i in 1:length(total_sorted)) {
    cat("\n",i, total_sorted[i])
}


 1 52
 2 67
 3 68
 4 77
 5 79
 6 94
 7 105
 8 124
 9 125
 10 126
 11 126
 12 126
 13 127
 14 127
 15 128
 16 129
 17 131
 18 132
 19 133
 20 134

In [34]:
tab <- matrix(nrow=4, ncol=11, byrow=TRUE)
rownames(tab) <- c('CE-IQ', 'CE-Rank', 'EE-IQ', 'EE-Rank')
colnames(tab)[11] <- 'TOTAL'
tab[1,] = c(CE_IQ, NA)
tab[2,] = c(9, 13.5, 11, 11, 7, 15, 13.5, 11, 5, 8, 104)
tab[3,] = c(EE_IQ, NA)
tab[4,] = c(17, 16, 4, 1, 20, 19, 6, 3, 2, 18, 106)
tab

<NA> <NA>  <NA> <NA> <NA> <NA> <NA>  <NA> <NA> <NA> TOTAL
CE-IQ   125  127.0 126  126  105  128  127.0 126  79   124   NA  
CE-Rank   9   13.5  11   11    7   15   13.5  11   5     8  104  
EE-IQ   131  129.0  77   52  134  133   94.0  68  67   132   NA  
EE-Rank  17   16.0   4    1   20   19    6.0   3   2    18  106

## c)

In [35]:
U_CE = 10*10 + 10*(10+1)/2 - 104
U_CE

[1] 51

In [36]:
U_EE = 10*10 + 10*(10+1)/2 - 106
U_EE

[1] 49

In [37]:
U_Stat = min(U_CE, U_EE)
U_Stat

[1] 49

![mannwhitneytwotailed2](img/mannwhitneytwotailed2.png)

**We can see that $U_{crit} = 23$ Here!**

**And $U_{crit} < U_{stat}$ at $\alpha = 0.05$**

**So we CAN'T reject the null hypothesis and 2 groups are approximately similiar**

<hr/>

**We can use normal distribution for p_value in this question**

In [39]:
Mu_U = 10*10 / 2
Mu_U

[1] 50

In [40]:
std_err = sqrt((10*10 + (10+10+1))/12)
std_err

[1] 3.175426

In [41]:
z = (U_Stat - Mu_U)/std_err
z

[1] -0.3149183

**after calculation we get that The P-Value is .753521.**

**and therefor The result is not significant at p < .05**

**So we CAN'T reject the null hypothesis and 2 groups are approximately similiar**

## d)

In [54]:
2*pnorm(q=-0.314)

[1] 0.753521

## e)

as said before The p-value is 0.7535. If we use a significance level of α = 0.05, we would fail to reject the null hypothesis of our hypothesis test because this p-value is not less than 0.05. therefor the sum of ranks of groups are the same and data in groups are similiar